classes/Project/Project.mjs

```diff
@@ -56,7 +93,12 @@ export default class Project {
     if (this.projectData.creator === userAgent) {
       return {
         hasAccess: true,
-        message: "User is the creator of the project and has access."
+        permissions: {
```

Comment:
Ultimately, I think this will be changed to the OWNER Role, so the `creator` can "give away" a project if desired.